src/main/java/org/broadinstitute/hellbender/tools/evoquer/GnarlyGenotyperEngine.java

```diff
@@ -0,0 +1,492 @@
+package org.broadinstitute.hellbender.tools.evoquer;
```

Comment:
This class should really not be in the `evoquer` package -- can you create a `gnarlygenotyper` package, and move both this class and the `GnarlyGenotyper` itself there?

Reply:
In that case, you're going to need to move your GnarlyGenotyperEngine as well.

Reply:
That's fine -- my copy will be deleted once I start using yours.